package.json

```diff
@@ -24,12 +24,12 @@
     "mout": "0.x"
   },
   "devDependencies": {
-    "mocha": "1.x",
-    "should": "1.x",
-    "jshint": "2.x",
+    "@dynamicaction/eslint-config-dna": "^2.0.0-pre.2",
```

Comment:
We have to make that repo public

Reply:
why do we even have a dynamicaction fork of flok? :thinking:

Reply:
I'm making the changes here and suggests PR to the `seangarner/flok` after its being approved.

Reply:
I don't think I want to publicly publish eslint config.  I'm not sure I want to add a dep to it from flok either.  I almost certainly wouldn't use it from my other open source projects for example - I'd cut a new custom eslint so I could control changes myself rather than link my OS efforts to dna.